android/src/main/java/io/mosip/tuvali/openid4vpble/Openid4vpBleModule.kt

```diff
@@ -119,13 +120,14 @@ class Openid4vpBleModule(private val reactContext: ReactApplicationContext) :
   fun destroyConnection(callback: Callback) {
     //TODO: Make sure callback can be called only once[Can be done once wallet and verifier split into different modules]
     Log.d(logTag, "destroyConnection called at ${System.nanoTime()}")
-
-    stopBLE(callback)
+    safeExecute.run {
```

Comment:
This one seems redundant. `stopBLE` already handling exception below

Reply:
as this is public function, I feel it would be safer to keep safeExecute as people might add some more lines of code later and forget to add safe Execute

And StopBLE is called in multiple places so added safe execute for safety